cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,70 @@
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
+import HomeAndCataloguePageObject
+  from '../support/pages/homeСatalogue.pageObject';
+import faker from 'faker';
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+const contactForm = new ContactFormPageObject();
+const homePage = new HomeAndCataloguePageObject();
+
+const testData = {
+  email: faker.internet.email(),
+  name: faker.name.firstName(),
+  city: faker.random.words(),
+  successMessage: 'Product added',
+  country: faker.random.words(),
+  month: faker.random.word(),
+  purchaseMessage: 'Thank you for your purchase!'
+
+};
 
+describe('Contact', () => {
+  before(() => {
+    homePage.visit();
   });
 
-  it('', () => {
+  it('should provide the ability to send feedback', () => {
```

Comment:
this spec checks not the ability to send feedback. You are checking purchase here. Review the test description